src/client/datascience/insidersNativeNotebookSurveyBanner.ts

```diff
@@ -127,8 +127,8 @@ export class InsidersNativeNotebooksSurveyBanner implements IExtensionSingleActi
         this.browserService.launch('https://aka.ms/vscjupyternb');
     }
 
-    private async isInsidersNativeNotebooksUser() {
-        return this.useVSCodeNotebookEditorApi;
+    private isInsidersNativeNotebooksUser() {
+        return this.useVSCodeNotebookEditorApi && this.applicationEnvironment.channel === 'insiders';
```

Comment:
Fixed the assumpgion, the prompt is required only for Insider users. Previously the experiment was enabled only in insiders, hence that condition worked, but not anymore (not if native is available for Stable users as well).

Reply:
Do we need to update dataScienceSurveyBanner as well to ensure we don't show prompts if the user has hit the threshold for usage of both old and native notebooks? This could happen if the user was on old notebooks before, hit the threshold, but ignored our prompts (in which case the agreed behavior was to keep showing them until they click either 'Yes' or 'No'), then got opted into native notebooks and also hit the threshold there.

I think you mentioned that the plan is to only show native notebooks to new users, but if I misremembered, this scenario could occur.

Reply:
> plan is to only show native notebooks to new users

Nope, native notebooks will be for both existing & new users.